README.md

```diff
@@ -4,19 +4,33 @@ When creating a [Google Cloud Dataproc](https://cloud.google.com/dataproc/) clus
 
 ## How initialization actions are used
 
-Initialization actions are stored in a [Google Cloud Storage](https://cloud.google.com/storage) bucket and can be passed as a parameter to the `gcloud` command or the `clusters.create` API when creating a Cloud Dataproc cluster. For example, to specify an initialization action when creating a cluster with the `gcloud` command, you can run:
+Initialization actions should be stored in a [Google Cloud Storage](https://cloud.google.com/storage) bucket and can be passed as a parameter to the `gcloud` command or the `clusters.create` API when creating a Cloud Dataproc cluster. For example, to specify an initialization action when creating a cluster with the `gcloud` command, you can run:
 
-    gcloud dataproc clusters create <CLUSTER_NAME> \
-      [--initialization-actions [GCS_URI,...]] \
-      [--initialization-action-timeout TIMEOUT]
+```bash
+gcloud dataproc clusters create <CLUSTER_NAME> \
+    [--initialization-actions [GCS_URI,...]] \
+    [--initialization-action-timeout TIMEOUT]
+```
+
+During development, you can create Dataproc cluster using Dataproc-provided regional initialization actions buckets:
+
+```bash
+REGION=<region>
+CLUSTER=<cluster_name>
+gcloud dataproc clusters create ${CLUSTER} \
+    --region ${REGION} \
+    --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/presto/presto.sh
```

Comment:
I've notived the word region confuses some people and they keep trying "us-east1-a" for instance. Consider adding an example of a region name or gcloud command to list regions

Reply:
Added full bucket name example and more information about Dataproc regional endopints to top-level README.md